Mathlib/Data/Sum/Interval.lean

```diff
@@ -209,4 +317,110 @@ theorem Ioo_inr_inr : Ioo (inr b₁ : Sum α β) (inr b₂) = (Ioo b₁ b₂).ma
 
 end Disjoint
 
+/-! ### Lexicographical sum of orders -/
+
+namespace Lex
+variable [Preorder α] [Preorder β] [OrderTop α] [OrderBot β] [LocallyFiniteOrder α]
+  [LocallyFiniteOrder β]
+
+/-- Throwaway tactic. -/
+private unsafe def simp_lex : tactic Unit :=
+`[refine to_lex.surjective.forall₃.2 _, rintro (a | a) (b | b) (c | c); simp only
```

Comment:
```suggestion
  `[refine to_lex.surjective.forall₃.2 _, rintro (a | a) (b | b) (c | c); simp only
```

Reply:
Though I suspect you'll need `?_` not `_`

Reply:
Something is wrong with the syntax...